tests/integration-tests/pom.xml

```diff
@@ -177,6 +178,11 @@
       <!-- END MQTT Deps -->
 
       <dependency>
+         <groupId>org.awaitility</groupId>
+         <artifactId>awaitility</artifactId>
+         <version>2.0.0</version>
+      </dependency>
+      <dependency>
```

Comment:
Rather than introducing new deps, you can re-use the wait utility we already in the tests.  I'll add an example.